src/tabs/tab-header-bar.tsx

```diff
@@ -194,13 +201,18 @@ export function TabHeaderBar({
     function isElementRegistered(element: HTMLElement) {
       return navigationAPI.current?.isRegistered(element) ?? false;
     }
-    function isElementDisabled(element: HTMLElement) {
+    function isElementFocusable(element: HTMLElement) {
```

Comment:
The function name suggests it returns `true` when the tab is focusable but it does the opposite. Either the function name or the logic inside should be changed.

Reply:
Negate the logic inside the function and left name as `isElementFocusable`. Moreover, added new conditions, because the recently merged feature "tabs with actions" changed the rules